blocks.py

```diff
@@ -18,4 +18,24 @@ def gyro_straight(tank, gyro, millies, target):
         subtract = gyro.angle() - target  # subtracts color sensors
         multiply = subtract * SHARPNESS # multiplies the difference by 0.7
         tank.drive(SPEED, multiply) # follows the line
+        
+        
+def stop_on_black(left_cs, right_cs):
+        if left_cs.reflection() <= 14:
+            return True
+        else:
+            return False
 
+        
+def perpendicular_line(tank, left_cs, right_cs, line_distance, left_wheel, right_wheel):
```

Comment:
Where do we use `left_wheel` and `right_wheel`?
```suggestion
def perpendicular_line(tank, left_cs, right_cs, line_distance):
```